src/mame/drivers/midqslvr.cpp

```diff
@@ -438,64 +667,58 @@ void midqslvr_state::midqslvr(machine_config &config)
 }
 
 
-ROM_START( offrthnd )
+ROM_START( hydrthnd )
 	ROM_REGION32_LE(0x80000, "bios", 0)
 	ROM_LOAD( "lh28f004sct.u8b1", 0x000000, 0x080000, CRC(ab04a343) SHA1(ba77933400fe470f45ab187bc0d315922caadb12) )
 
-	ROM_REGION( 0x8000, "video_bios", ROMREGION_ERASEFF ) // TODO: no VGA card is hooked up, to be removed
-//  ROM_LOAD16_BYTE( "trident_tgui9680_bios.bin", 0x0000, 0x4000, BAD_DUMP CRC(1eebde64) SHA1(67896a854d43a575037613b3506aea6dae5d6a19) )
-//  ROM_CONTINUE(                                 0x0001, 0x4000 )
+	ROM_REGION( 0x40000, "iocpu", 0 )   /* Diego board CY7C63513-PVC MCU code */
+	ROM_LOAD( "diego.u8", 0x000000, 0x040000, NO_DUMP ) // 8KB internal EPROM
 
 	DISK_REGION( "ide:0:hdd:image" )
-	DISK_IMAGE( "offrthnd", 0, SHA1(d88f1c5b75361a1e310565a8a5a09c674a4a1a22) )
+	DISK_IMAGE( "hydro", 0,  SHA1(d481d178782943c066b41764628a419cd55f676d) )
 ROM_END
 
-ROM_START( hydrthnd )
+ROM_START( offrthnd )
 	ROM_REGION32_LE(0x80000, "bios", 0)
 	ROM_LOAD( "lh28f004sct.u8b1", 0x000000, 0x080000, CRC(ab04a343) SHA1(ba77933400fe470f45ab187bc0d315922caadb12) )
 
-	ROM_REGION( 0x8000, "video_bios", ROMREGION_ERASEFF ) // TODO: no VGA card is hooked up, to be removed
-//  ROM_LOAD16_BYTE( "trident_tgui9680_bios.bin", 0x0000, 0x4000, BAD_DUMP CRC(1eebde64) SHA1(67896a854d43a575037613b3506aea6dae5d6a19) )
-//  ROM_CONTINUE(                                 0x0001, 0x4000 )
+	ROM_REGION( 0x40000, "iocpu", 0 )   /* Magicbus board CY7C63513-PVC MCU code */
+	ROM_LOAD( "magicbus.u18", 0x000000, 0x040000, NO_DUMP ) // 8KB internal EPROM
```

Comment:
8KiB is 0x2000 bytes - you've got 32 times the size here.

Reply:
Done